### PR TITLE
Ensure control built-in shortcuts are matched exactly & add shortcuts for SpriteFrames editor

### LINKED ISSUE #63804: Pressing delete after selecting a SpriteFrame animation frame deletes the selected node
### Godot version

v4.0.alpha.calinou [8243c7ab5]

### System information

Windows 10

### Issue description

[Preview](https://i.imgur.com/A33hVvv.mp4)

Pressing the delete key after selecting a SpriteFrame animation frame deletes the selected node instead of the selected animation frame. The delete key should be context-aware and should always delete the last item the user has selected.

This seems to be working fine in the FileSystem panel where it deletes the selected file instead of the node in the scene.

### Steps to reproduce

1. Create a new project.
2. Create a new SpriteFrame resource.
3. Add animation frames to the SpriteFrame resource.
4. Create a new scene with an AnimatedSprite2D node.
5. Assign the SpriteFrame resource to the AnimatedSprite2D node.
6. Select a frame in the SpritFrames panel.
7. Press the delete key.
8. A prompt saying "Delete node" appears.
9. Press OK.
10. The node is deleted in the scene.

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by EricEzaM**
Closes #63804

I had to make the changes contained in the first commit to `ItemList` because otherwise shortcuts like Ctrl+Right/Left and Alt+Right/Left would not work, as the ItemList would swallow them as `ui_right` or `ui_left`. All built-in control shortcuts which are based on actions should use `exact_match` - so I made this change for all cases.

Also some minor renaming for clarity and consistency inside the spriteframes editor code.

### CODE REVIEW

File: editor/plugins/sprite_frames_editor_plugin.cpp [hunk 1]
```diff
@@ -1187,18 +1187,16 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	HBoxContainer *hbc_animlist = memnew(HBoxContainer);
 	sub_vb->add_child(hbc_animlist);
 
-	new_anim = memnew(Button);
-	new_anim->set_flat(true);
-	new_anim->set_tooltip_text(TTR("New Animation"));
```
Comment (KoBeWi) [inline editor/plugins/sprite_frames_editor_plugin.cpp:1192]: What happened to the tooltip?

File: editor/plugins/sprite_frames_editor_plugin.cpp [hunk 2]
```diff
@@ -1333,13 +1327,38 @@ SpriteFramesEditor::SpriteFramesEditor() {
...
+	empty_before->set_shortcut(ED_SHORTCUT("sprite_frames/empty_before", TTR("Insert Empty (Before Selected)"), KeyModifierMask::ALT | Key::LEFT));
+	empty_after->set_shortcut_context(tree);
+	empty_after->set_shortcut(ED_SHORTCUT("sprite_frames/empty_after", TTR("Insert Empty (After Selected)"), KeyModifierMask::ALT | Key::RIGHT));
+	move_up->set_shortcut_context(tree);
+	move_up->set_shortcut(ED_SHORTCUT("sprite_frames/move_left", TTR("Move Frame Left"), KeyModifierMask::CMD_OR_CTRL | Key::LEFT));
+	move_down->set_shortcut_context(tree);
+	move_down->set_shortcut(ED_SHORTCUT("sprite_frames/move_right", TTR("Move Frame Right"), KeyModifierMask::CMD_OR_CTRL | Key::RIGHT));
+
+	zoom_out->set_shortcut_context(tree);
+	zoom_out->set_shortcut(ED_SHORTCUT("sprite_frames/zoom_out", TTR("Zoom Out"), KeyModifierMask::CMD_OR_CTRL | Key::MINUS));
```
Comment (KoBeWi) [inline editor/plugins/sprite_frames_editor_plugin.cpp:1358]: Shouldn't the shortcut include `KP_MINUS` too?

Comment (EricEzaM) [reply editor/plugins/sprite_frames_editor_plugin.cpp:1358]: Yes, why not

**Review by KoBeWi — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**